test/commands/unmarshalPassthrough.js

```diff
@@ -1,23 +1,27 @@
 'use strict';
 
-var unmarshalPassThrough = require('../../lib/commands/unmarshalPassThrough'),
-    withData = require('leche').withData;
+var unmarshalPassThrough = require('../../lib/commands/unmarshalPassThrough');
+var withData = require('leche').withData;
 
 describe('unmarshalPassThrough', function() {
-    it('should return undefined if item does not contain a key which is a pass through type', function() {
-        var result = unmarshalPassThrough({FOO: 'bar'});
-        (result === undefined).should.equal(true);
-    });
+  it(
+    'should return undefined if item does not contain a key which is a ' +
+    'pass through type',
+    function() {
+      var result = unmarshalPassThrough({FOO: 'bar'});
+      (result === undefined).should.equal(true);
+    }
+  );
 
-    withData({
-        'with an "S" item': [{S: 'foo'}, 'foo'],
-        'with "SS" item': [{SS: ['foo', 'bar']}, ['foo', 'bar']],
-        'with "B" item': [{B: 'binary'}, 'binary'],
-        'with "BS" item': [{BS: ['binary1', 'binary2']}, ['binary1', 'binary2']],
-        'with "BOOL" item': [{BOOL: true}, true]
-    }, function(item, expected) {
-        it('should return the value for the pass through key', function() {
-            unmarshalPassThrough(item).should.eql(expected);
-        });
+  withData({
+    'with an "S" item': [{S: 'foo'}, 'foo'],
+    'with "SS" item': [{SS: ['foo', 'bar']}, ['foo', 'bar']],
+    'with "B" item': [{B: 'binary'}, 'binary'],
+    'with "BS" item': [{BS: ['binary1', 'binary2']}, ['binary1', 'binary2']],
+    'with "BOOL" item': [{BOOL: true}, true]
```

Comment:
withData adds `with` to the name of the test automatically so these test results are outputting `with with an "s" item`, not a big deal but makes reading the test results a little odd.

From travis:
```
unmarshalPassThrough
  ✓ should return undefined if item does not contain a key which is a pass through type 
    with with an "S" item
  ✓ should return the value for the pass through key 
    with with "SS" item
  ✓ should return the value for the pass through key 
    with with "B" item
  ✓ should return the value for the pass through key 
    with with "BS" item
  ✓ should return the value for the pass through key 
    with with "BOOL" item
  ✓ should return the value for the pass through key 
```